include/ShowerParametrization.h

```diff
@@ -0,0 +1,101 @@
+
```

Comment:
File to be removed. Now the directory is named `interface` to be compatible with CMSSW convention